api-spec/definitions/WFS3core.fragment.yaml

```diff
@@ -0,0 +1,385 @@
+        links:
+          type: array
+          items:
+            $ref: '#/components/schemas/link'
+          example:
+            - href: 'http://data.example.org/collections/buildings/items'
+              rel: item
+              type: application/geo+json
+              title: Buildings
+            - href: 'http://example.org/concepts/building.html'
+              rel: describedBy
+              type: text/html
+              title: Feature catalogue for buildings
+        extent:
+          $ref: '#/components/schemas/extent'
+        crs:
+          description: >-
+            The coordinate reference systems in which geometries may be
+            retrieved. Coordinate reference systems are identified by a URI. The
+            first coordinate reference system is the coordinate reference system
+            that is used by default. This is always
+            "http://www.opengis.net/def/crs/OGC/1.3/CRS84", i.e. WGS84
+            longitude/latitude.
+          type: array
+          items:
+            type: string
+          default:
+            - 'http://www.opengis.net/def/crs/OGC/1.3/CRS84'
+    extent:
+      type: object
+      properties:
+        crs:
+          description: >-
+            Coordinate reference system of the coordinates in the spatial extent
+            (property `spatial`). In the Core, only WGS84 longitude/latitude is
+            supported. Extensions may support additional coordinate reference
+            systems.
+          type: string
+          enum:
+            - 'http://www.opengis.net/def/crs/OGC/1.3/CRS84'
+          default: 'http://www.opengis.net/def/crs/OGC/1.3/CRS84'
+        spatial:
+          description: >-
+            West, north, east, south edges of the spatial extent. The minimum
+            and maximum values apply to the coordinate reference system WGS84
+            longitude/latitude that is supported in the Core. If, for example, a
+            projected coordinate reference system is used, the minimum and
+            maximum values need to be adjusted.
+          type: array
+          minItems: 4
+          maxItems: 6
+          items:
+            type: number
+          example:
+            - -180
+            - -90
+            - 180
+            - 90
+        trs:
+          description: >-
+            Temporal reference system of the coordinates in the temporal extent
+            (property `temporal`). In the Core, only the Gregorian calendar is
+            supported. Extensions may support additional temporal reference
+            systems.
+          type: string
+          enum:
+            - 'http://www.opengis.net/def/uom/ISO-8601/0/Gregorian'
+          default: 'http://www.opengis.net/def/uom/ISO-8601/0/Gregorian'
+        temporal:
+          description: Begin and end times of the temporal extent.
+          type: array
+          minItems: 2
+          maxItems: 2
+          items:
+            type: string
+            format: dateTime
```

Comment:
We inherited this problem from WFS (see https://github.com/opengeospatial/WFS_FES/issues/155 - fourth issue), but we may change this anyway from `dateTime` to `date-time`? According to specs the latter is correct. See also https://github.com/radiantearth/stac-spec/issues/146